src/tjlf_read_input.jl

```diff
@@ -191,4 +199,42 @@ function checkInput(inputTJLFVector::Vector{InputTJLF})
     for inputTJLF in inputTJLFVector
         checkInput(inputTJLF)
     end
-end
\ No newline at end of file
+end
+"""
+    save(input::Union{InputTJLF}, filename::AbstractString)
+
+Write input_tglf to file in input.tglf format to be read by TGLF
+"""
+
+function save(input::InputTJLF, filename::AbstractString)
```

Comment:
no space between the function and the docstring, otherwise the documentation does not pick it up

```suggestion
"""
    save(input::Union{InputTJLF}, filename::AbstractString)

Write input_tglf to file in input.tglf format to be read by TGLF
"""
function save(input::InputTJLF, filename::AbstractString)
```